src/libraries/Microsoft.PowerFx.Core/Errors/BaseError.cs

```diff
@@ -167,42 +167,13 @@ internal BaseError(IDocumentError innerError, Exception internalException, Docum
                 longMessage = errorResource.GetSingleValue(ErrorResource.LongMessageTag);
```

Comment:
Should this block also be pulled out to the utils function?
Something like:
`(string shortMessage, string longMessage) GetLocalizedErrorContent(errKey, locale, args)`